build: add support for multiple Github Templates (#112)
amp-labs/connectors: .github/PULL_REQUEST_TEMPLATE/new_connector.md

## Checklist
- [ ] Ran Linter
- [ ] Catalog tests passing

> - [ ] Created PR from non-main branch

Reviewer: This third item is probably not needed, Github wouldn't have let you create a PR from a main branch.

Reviewer 2: This was for forks, but is irrelevant now

## Catalog variables

> <Please mention any variables you had to use for the provider's info>

Reviewer: ```suggestion
<Does the provider's info include any template variables such as `{{workspace}}`?>
```
